strawberryfields/tdm/tdmprogram.py

```diff
@@ -447,10 +445,10 @@ def roll(self):
         self.circuit = self.rolled_circuit
         return self
 
-    def unroll(self):
+    def unroll(self, shots=1):
```

Comment:
Out of curiosity, how come a default value was added here for shots? Does it make more sense to have `unroll(self, shots)`, so that shots always have to be provided?

Reply:
I think I was just a bit quick adding this here. It makes sense letting it always having to be provided. I'll fix it!